flow/CMakeLists.txt

```diff
@@ -113,6 +113,9 @@ configure_file(${CMAKE_CURRENT_SOURCE_DIR}/config.h.cmake ${CMAKE_CURRENT_BINARY
 
 add_flow_target(STATIC_LIBRARY NAME flow SRCS ${FLOW_SRCS})
 target_link_libraries(flow PRIVATE stacktrace)
+if(NOT USE_ASAN AND NOT USE_VALGRIND AND NOT USE_TSAN AND NOT USE_UBSAN AND NOT USE_MSAN)
```

Comment:
Could we add a `USE_JEMALLOC` variable with this condition? It would be good to set the `WITH_JEMALLOC` variable in the RocksDB cmake explicitly. (I'm working on testing it with and without that configuration right now)

Reply:
sorry it's not quite clear to me what the ask is. Do you want a mechanism to disable usage of jemalloc? Or do you want a mechanism to use jemalloc even when building for sanitizer/valgrind? I think both are reasonable

Reply:
Ah, sorry. I'm looking for a way to tell RocksDB to do the same thing that FDB is doing. I'm looking for a variable that `CompileRocksDB.cmake` can read to know if jemalloc is being used by FDB.

Reply:
with this change fdb will always use jemalloc (except for Windows builds -- but I am not sure whether your rocks storage engine works on Windows? If it does you can use `WIN32`)

Reply:
I don't follow. Doesn't this line mean that we won't use jemalloc when ASAN/TSAN/etc are enabled?

I have no idea if the RocksDB engine works on WIndows.

Reply:
I stand corrected... This PR had been open long enough that I already forgot about some of the weirder stuff :)

I guess you could also check for sanitizers but that becomes unreasonable. I will add a variable. I assume you don't need this variable to be exported?

Reply:
I fixed this, PTAL

Reply:
Yep. That should work. Thanks!